libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -225,31 +216,22 @@ - (void)testMultipleAccounts {
         }
     }
     
-    // Remove and re-load all accounts
-    {
-        [self.uam clearAllAccountState];
-        XCTAssertEqual([self.uam.allUserIdentities count], (NSUInteger)0, @"There should be no accounts");
-
-        XCTAssertTrue([self.uam loadAccounts:&error], @"Unable to load user accounts: %@", error);
-        
-        for (NSUInteger index=0; index<10; index++) {
-            XCTAssertEqualObjects(((SFUserAccountIdentity *)self.uam.allUserIdentities[index]).userId, ([NSString stringWithFormat:kUserIdFormatString, (unsigned long)index]), @"User ID doesn't match");
-            XCTAssertEqualObjects(((SFUserAccountIdentity *)self.uam.allUserIdentities[index]).orgId, ([NSString stringWithFormat:kOrgIdFormatString, (unsigned long)index]), @"Org ID doesn't match");
-        }
-    }
-    
-    // Remove and verify that allUserAccounts property implicitly loads the accounts from disk.
+       // Remove and verify that allUserAccounts property implicitly loads the accounts from disk.
```

Comment:
Ah, here's where `clearAllAccountState` gets interesting.  This block of code is meant to test that accounts can be loaded properly from disk, and `clearAllAccountState` ensures that there are no accounts loaded _in memory_, but still retains them on disk.